website/pages/blog/announcing-cloudquery-aws-event-based-sync-beta.mdx

```diff
@@ -127,6 +127,6 @@ At the moment, only one Kinesis stream is supported by a running instance of Clo
 
 The current coverage of tables has been designed to provide a selection of different services. We will add more resources based on your feedback.
 
-[Read the docs](https://cloudquery.io/docs/plugins/sources/aws/event-based-sync#supported-services-and-events)
+[Read the docs](/docs/plugins/sources/aws/event-based-sync#supported-services-and-events)
```

Comment:
Changed this to a relative link to avoid it opening in a new tab and redirecting